.travis.yml

```diff
@@ -39,7 +39,9 @@ branches:
 install:
  - true
 script:
-  - ./gradlew ciBuild release
+  - TRAVIS_MESSAGE="$(git log --format=%B -n 1)"
+  - ./gradlew ciBuild
+  - if [[ $TRAVIS_MESSAGE != *"[ci skip-release]"* ]] ; then ./gradlew release; fi
```

Comment:
The git log command was from a travis issue as there is an intent to add the new environment variable. The command is kinda magic, so naming the variable makes it imo easier to comprehend.